frontend/smart-kickers-game/src/hooks/useHeatmap.js

```diff
@@ -0,0 +1,38 @@
+import useAxios from 'axios-hooks';
+import config from '../config';
+import { useMemo } from 'react';
+
+const useHeatmap = () => {
+  const [{ data, loading, error }] = useAxios({
+    method: 'get',
+    url: `${config.apiBaseUrl}/stats`,
+  });
+
+  const heatmap = useMemo(() => (data?.Heatmap ? mirrorHeatmap() : []), [data]);
+
+  function mirrorHeatmap() {
+    const heatmapDim = data.Heatmap.length;
+    const array = new Array(heatmapDim).fill(0).map(() => '');
```

Comment:
I believe you can fill it with `''`, then the `map` function won't be needed.

Reply:
Done